ui/src/containers/NodePageMetricsTab.js

```diff
@@ -103,31 +136,129 @@ const NodePageMetricsTab = (props) => {
     workloadPlaneNetworkBandwidthOut: 'out',
   };
 
+  const typedAvgMetrics = {
+    cpuUsage: 'cluster avg',
+    systemLoad: 'cluster avg',
+    memory: 'cluster avg',
+    iopsRead: 'read avg',
+    iopsWrite: 'write avg',
+    controlPlaneNetworkBandwidthIn: 'in avg',
+    controlPlaneNetworkBandwidthOut: 'out avg',
+    workloadPlaneNetworkBandwidthIn: 'in avg',
+    workloadPlaneNetworkBandwidthOut: 'out avg',
+  };
+
   const nodeStatsData = Object.keys(nodeStats).reduce((acc, metricName) => {
     const data = operateMetricRawData(nodeStats[metricName][0]?.values);
 
     let extra = {};
     const metricType = typedMetrics[metricName];
     if (metricType !== undefined) extra.type = metricType;
 
+    /*
+     ** Using 'symbol': 'A' because vega-lite internally assign the plain line
+     ** to the first alphabetical item when strokeDash is used
+     */
     acc[metricName] = data.map((slot) => ({
       date: fromUnixTimestampToDate(slot[0]),
-      [metricType]: slot[1],
+      y: slot[1],
+      symbol: 'A',
       ...extra,
     }));
     return acc;
   }, {});
 
+  const avgStatsData = Object.keys(avgStats).reduce((acc, metricName) => {
+    const data = operateMetricRawData(avgStats[metricName][0]?.values);
+
+    let extra = {};
+    const metricType = typedAvgMetrics[metricName];
+    if (metricType !== undefined) extra.type = metricType;
+
+    acc[metricName] = data.map((slot) => ({
+      date: fromUnixTimestampToDate(slot[0]),
+      y: slot[1],
+      symbol: 'Cluster avg',
+      ...extra,
+    }));
+    return acc;
+  }, {});
+
+  const tooltipConfigUsage = getTooltipConfig([
+    { field: '%', type: 'quantitative', title: '%', format: '.2f' },
+    showAvg && {
+      field: 'cluster avg',
+      type: 'quantitative',
+      title: 'Cluster avg',
+      format: '.2f',
+    },
+  ]);
+
+  let cpuData = nodeStatsData['cpuUsage'];
+  if (showAvg) cpuData = cpuData.concat(avgStatsData['cpuUsage']);
+
+  let systemLoadData = nodeStatsData['systemLoad'];
+  if (showAvg)
+    systemLoadData = systemLoadData.concat(avgStatsData['systemLoad']);
+
+  let memoryData = nodeStatsData['memory'];
+  if (showAvg) memoryData = memoryData.concat(avgStatsData['memory']);
+
+  const tooltipConfigIops = getTooltipConfig([
+    { field: 'read', type: 'quantitative', title: 'read', format: '.2f' },
```

Comment:
We need to make sure the display of the tooltip follows the design.

```suggestion
    { field: 'read', type: 'quantitative', title: `Read - ${nodeName}`, format: '.2f' },
```